lib/couchbase/protostellar/client.rb

```diff
@@ -34,95 +36,43 @@ def initialize(host, credentials, channel_args)
         @view_stub = Generated::View::V1::View::Stub.new(host, credentials, channel_override: @channel)
       end
 
-      def kv
-        @kv_stub
+      def stub(service)
+        case service
+        when :analytics
+          @analytics_stub
+        when :kv
+          @kv_stub
+        when :query
+          @query_stub
+        when :routing
+          @routing_stub
+        when :search
+          @search_stub
+        when :view
+          @view_stub
+        else
+          raise ArgumentError "service `#{service}' not recognised"
```

Comment:
This error is one of those that will not ever be raised unless we did something wrong, should this be a specific type of error?